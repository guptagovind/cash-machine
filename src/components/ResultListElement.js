import React from 'react';

const ResultListElement = ({result}) => (
          <span>{result !== "Empty Set" ? result.toFixed(2) : result}</span>
);

export default ResultListElement;
