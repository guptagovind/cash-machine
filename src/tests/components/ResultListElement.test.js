import React from 'react';
import {shallow} from 'enzyme';
import ResultListElement from '../../components/ResultListElement';


let wrapper;
beforeEach(() => {
  const result=100;
  wrapper = shallow(<ResultListElement result={result}/>);
});

test('should render the resultsList', () => {
  expect(wrapper).toMatchSnapshot();
});
