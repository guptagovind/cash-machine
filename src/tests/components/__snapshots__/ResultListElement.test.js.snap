// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the resultsList 1`] = `
<span>
  100.00
</span>
`;
